core/src/test/java/org/apache/iceberg/TestSequenceNumberForV2Table.java

```diff
@@ -319,12 +321,18 @@ public void testExpirationInTransaction() {
     V2Assert.assertEquals("Snapshot sequence number should be 2", 2, snap2.sequenceNumber());
     V2Assert.assertEquals(
         "Last sequence number should be 2", 2, readMetadata().lastSequenceNumber());
+    V2Assert.assertEquals(
+        "Should be 2 snapshot files", listSnapshotFiles(table.location()).size(), 2);
 
     Transaction txn = table.newTransaction();
     txn.expireSnapshots().expireSnapshotId(commitId1).commit();
     txn.commitTransaction();
     V2Assert.assertEquals(
         "Last sequence number should be 2", 2, readMetadata().lastSequenceNumber());
+    V2Assert.assertEquals(
+        "Should be 1 snapshot file as 1 was deleted",
```

Comment:
Nit: Typically we use the term "manifest list" instead of  "snapshot file" could we call it that?